core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -1217,9 +1224,44 @@ class UserServiceSpec
 
   behavior of "getBillingProject"
 
+  it should "return the project when it exists" in {
+    val projectId = UUID.randomUUID()
+    val projectName = RawlsBillingProjectName(UUID.randomUUID().toString)
+    val project = RawlsBillingProject(projectId, projectName, CreationStatuses.Ready, None, None)
+    val repository = mock[BillingRepository]
+    when(repository.getBillingProjectById(projectId)).thenReturn(Future.successful(Some(project)))
+    when(repository.getBillingProject(projectName)).thenReturn(Future.successful(Some(project)))
```

Comment:
Nit: this second mock isn't needed

Reply:
Fixed: https://github.com/broadinstitute/rawls/pull/3205/commits/d5bfaad39e058c8e98bcd7c1f49ec543c0ed6f5a